components/list/list-item-drag-drop-mixin.js

```diff
@@ -606,6 +615,11 @@ export class NewPositionEventDetails {
 			}
 		}
 		list[destination] = item;
+
+		return {
+			origin: origin,
+			destination: destination
+		}
```

Comment:
We won't need this anymore if we are asking the parent for information on `focusin`.